docs/docs/guides/deploy/cloud.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: ZITADEL Cloud
+---
+
+## Get started
+
+<!-- TODO Insert Fabiennes get started here -->
```

Comment:
I am not sure if thats correct. My getting startet is about the SaaS Servie. Customer Portal, Create first Instance, etc. I would not search for saas service in the deployment section

Reply:
At the moment this contains mutliple guides in the basics. starting with the get started (customer portal), then instances, organisation, projects, etc.

Reply:
I think the get to know ZITADEL should only reflect how to use it not how to deploy it.
Otherwise it feels weird to me. Because self-hosting customers do not need to create a customerportal account and so on.

In my eyes our cloud is a legit deployment option. (Although this is somewhat influenced by how, Grafana and Cockroach structure there docs.)